utils/manage-api-users.ts

```
@@ -28,133 +30,144 @@ program
-        team_slug,
-        per_page: 500
-    })
-    data.forEach((user) => member += `${user.login}\n`)
+  .action(async () => {
+    let member = ``;
+    const { data, headers } = await ghApi.teams.listMembersInOrg({
+      org,
+      team_slug,
+      per_page: 500,
+    });
+    data.forEach((user) => member += `${user.login}\n`);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)